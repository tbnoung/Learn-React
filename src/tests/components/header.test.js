import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import Header from '../../components/Header'

test('render header', ()=>{
  const renderer = new ShallowRenderer();
  renderer.render(<Header/>)
  expect(renderer.getRenderOutput()).toMatchSnapshot();
})