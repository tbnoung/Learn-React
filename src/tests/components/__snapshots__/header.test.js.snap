// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render header 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <Unknown
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Dashboard
  </Unknown>
  <Unknown
    activeClassName="is-active"
    to="/create"
  >
    Create
  </Unknown>
  <Unknown
    activeClassName="is-active"
    to="/edit"
  >
    Edit
  </Unknown>
  <Unknown
    activeClassName="is-active"
    to="/help"
  >
    Help
  </Unknown>
</header>
`;
